site/specs/webrtc.yml

```diff
@@ -2421,6 +2520,201 @@ components:
+      type: object
+      description: The state data associated with a Dialog - dependent on the dialogType
+      required:
+        - to
+        - from
+      properties:
+        to:
+          type: string
+          description: the destination telephone number in e.164 format
+          example: '+16136211234'
+        from:
+          type: string
+          description: >-
+            the caller's telephone number in e.164 format, or the string UNKNOWN
+            or RESTRICTED
+          example: '+16136211234'
+        userToUser:
+          type: string
+          description: >
+            Information that will be handed to the SIP client to be passed
+            towards the far end user. The formatting of the contents is
+            completely dependent on the ability of the  end system to parse and
+            understand the contents
+          example: >-
+            eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c;encoding=jwt
+    sipDialogResponseStateData:
+      type: object
+      description: >
+        The dialogStateData contains the information that represents the current
+        state of the dialog.  Depending on the characteristics and state of the
+        Dialog this information may be missing,  current, out of date, or
+        idempotent.  Please see the Text documentation about the use of Dialogs
+        for further information.
+      properties:
+        state:
+          type: string
+          description: the name of the state that the Dialog has entered
+          example: ANSWERED
+          enum:
+            - NEW
+            - CALLING
+            - ANSWERED
+            - RINGING
+            - TERMINATED
+        to:
+          type: string
+          description: the destination telephone number in e.164 format
+          example: '+16136211234'
+        from:
+          type: string
+          description: >-
+            the caller's telephone number in e.164 format, or the string UNKNOWN
+            or RESTRICTED
+          example: '+16136211234'
+        userToUser:
+          type: string
+          description: >
+            Information that will be handed to the SIP client to be passed
+            towards the far end user. The formatting of the contents is
+            completely dependent on the ability of the  end system to parse and
+            understand the contents
+          example: >-
+            eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c;encoding=jwt
+    sipUpdateDialogRequestBody:
+      type: object
+      description: Update a SIP Dialog owned by a Participant.
+      properties:
+        state:
+          type: string
+          description: the name of the state that the Dialog has entered
+          example: ANSWERED
+        tag:
```

Comment:
This tag property does not exist in the create dialog request. Either we add it to the create endpoint (we'll need to talk about implementing it) or we remove it from here

Reply:
deletion is easier and makes more sense.